docs/examples/abac/nodejs/01-simple.js

```diff
@@ -0,0 +1,92 @@
+const { Oso } = require('oso');
+
+const oso = new Oso();
+
+const EXPENSES = [
+  { amount: 500, submitted_by: 'alice', location: 'NYC', project_id: 2 },
```

Comment:
Is it weird to use camel case in JavaScript?

Reply:
camelCase is idiomatic; snake_case is not

Reply:
Okay, there are a few other places where snake case is used in the JS examples, but I assume that's just for consistency with the rest of the docs for now. We need to go through all the docs for this issue anyway, can be a follow-up.

Reply:
> Okay, there are a few other places where snake case is used in the JS examples, but I assume that's just for consistency with the rest of the docs for now. We need to go through all the docs for this issue anyway, can be a follow-up.

assumption is correct; agree on the follow-up